grammar.js

```diff
@@ -282,12 +282,15 @@ module.exports = grammar({
       ']'
     ),
 
-    meta_item: $ => seq(
-      $.identifier,
-      optional(choice(
-        seq('=', $._literal),
-        seq('(', sepBy(',', $.meta_item), optional(','), ')')
-      ))
+    meta_item: $ => choice(
+      $._path,
+      seq(
+        $.identifier,
+        optional(choice(
+          seq('=', $._literal),
+          seq('(', sepBy(',', $.meta_item), optional(','), ')')
+        ))
+      )
```

Comment:
From the language spec, it looks like the literal expression and arguments can also be specified when a path is used in place of an identifier. Can you try updating this to reflect that possibility?

Reply:
sure